### DISCUSSION TITLE
Is there a command to pre-download models to local?

### DISCUSSION THREAD

**Discussion by SimZhou** [openai/whisper#2104] [Q&A]
Hi, I am trying to make a docker image to use Whisper, So I need to pre-download those models into the docker images when building the Dockerfile.

is there a single command I could use to download those models?

**Comment by ryanheise** [comment]
Answer in https://github.com/openai/whisper/pull/789#issuecomment-1387633017

There was a PR to add a command, but to keep the repo minimal, it was recommended to do something like this:

```bash
python -c "import whisper; print(whisper._download(whisper._MODELS['tiny'], '$HOME/.cache/whisper', False))"
```

(Assuming you're on Linux.)
